x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/test_pipeline.test.tsx

```diff
@@ -114,23 +115,24 @@ describe('Test pipeline', () => {
       expect(exists('outputTabContent')).toBe(true);
 
       // Click reload button and verify request
-      const totalRequests = server.requests.length;
       await actions.clickRefreshOutputButton();
       // There will be two requests made to the simulate API
       // the second request will have verbose enabled to update the processor results
-      expect(server.requests.length).toBe(totalRequests + 2);
-      expect(server.requests[server.requests.length - 2].url).toBe(
-        '/api/ingest_pipelines/simulate'
+      expect(httpSetup.post).toHaveBeenLastCalledWith(
+        `${API_BASE_PATH}/simulate`,
+        expect.anything()
       );
-      expect(server.requests[server.requests.length - 1].url).toBe(
-        '/api/ingest_pipelines/simulate'
+      expect(httpSetup.post).toHaveBeenLastCalledWith(
+        `${API_BASE_PATH}/simulate`,
+        expect.anything()
```

Comment:
nit: I don't think that would test for 2 different calls to the simulate API. You might want to use `toHaveBeenCalledTimes` or `toHaveBeenNthCalledWith`

Reply:
nice catch!, ill patch it up!